src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -518,21 +586,20 @@ $spacing: 4px;
 	&__label {
 		display: flex;
 		align-items: center;
+		flex-direction: row;
+		gap: 4px;
 		user-select: none;
 		min-height: $clickable-area;
 		border-radius: $clickable-area;
 		padding: 4px $icon-margin;
-		margin: 0 #{-$icon-margin};
+		width: 100%;
```

Comment:
```suggestion
```

This makes the buttons expand, which is different than before.

Reply:
Yes, but required for the text overflow on the button style. I fixed this by changing it to:
```css
width: 100%;
max-width: fit-content;
```
So it works for both: button style and radio / checkbox stye

Reply:
I thought I tested all version... Sorry :see_no_evil:

Reply:
It is still there, on the latest commit:
![image](https://github.com/nextcloud/nextcloud-vue/assets/93392545/3a820846-20ae-4728-8769-f326fba5de51)